apps/arm/ClawController.cpp

```diff
@@ -78,6 +78,14 @@ mbed_error_status_t ClawController::retractToolTip() {
     return err_status;
 }
 
+mbed_error_status_t ClawController::update(){
```

Comment:
This just kills the motor and disables the claw, you wont even be able to reverse the motor to remove the pressure. What you should really be doing is only allow the motor to be driven in reverse while this max force is exceeded. You should also play nicely with the ActuatorController lib and limit using the control scheme depending on the control mode

Reply:
Take a look at the ClawController update function now, how should I handle the position control mode? Does this mode even apply to controlling the claw?